### PR TITLE
Jenkins: Utilize jabba to select jdks used in tests

### PR DESCRIPTION
**Pull request by mfussenegger**
## Checklist

 - [x] User relevant changes are recorded in ``CHANGES.txt``
 - [x] Touched code is covered by tests
 - [x] Documentation has been updated if necessary
 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed
 - [x] This does not contain breaking changes, or if it does:
    - It is released within a major release
    - It is recorded in ``CHANGES.txt``
    - It was marked as deprecated in an earlier release if possible
    - You've thought about the consequences and other components are adapted
      (E.g. AdminUI)

### CODE REVIEW

**Comment by kovrus** [comment]
do you think it worth defining jdk versions in env variables 
```
pipeline {
    environment {
        JDK_12=...
...
```
and accessing it in stages?

File: Jenkinsfile [hunk 1]
```diff
@@ -16,13 +16,11 @@ pipeline {
...
           }
-          tools {
-            jdk 'jdk11'
-          }
           steps {
             sh 'git clean -xdff'
             checkout scm
-            sh 'timeout 30m ./gradlew --no-daemon --parallel -PtestForks=8 test forbiddenApisMain jacocoReport'
+            sh 'jabba install openjdk@1.11.0-2'
+            sh 'JAVA_HOME=$(jabba which --home openjdk@1.11.0-2) timeout 30m ./gradlew --no-daemon --parallel -PtestForks=8 test forbiddenApisMain jacocoReport'
```
Comment (seut) [inline Jenkinsfile]: isn't `jabba install` already setting `JAVA_HOME`?

Comment (mfussenegger) [reply Jenkinsfile]: No it doesn't. There would be `jabba use` which is supposed to set it, but it doesn't seem to work either.

Comment (seut) [reply Jenkinsfile]: Ok strange, on my machine its adjusted after an `jabba install?...

Comment (mfussenegger) [reply Jenkinsfile]:
```
[admin@jenkins-09 ~]$ jabba install openjdk@1.13.0-21 && echo $JAVA_HOME

[admin@jenkins-09 ~]$ 
```

**Pull request merged**